File: MarsRoverProb/src/com/marsroverprob/commands/RotateLeftCommand.java
package com.marsroverprob.commands;

import com.marsroverprob.helper.RoverHelper;
import com.marsroverprob.model.Rover;

public class RotateLeftCommand implements RoverCommand {

  public void execute(Rover rover, String maxPos) {
    RoverHelper.rotateLeft(rover, maxPos);
  }

}

File: MarsRoverProblem_C#/Source/marsroverprob/commands/RotateLeftCommand.cs
using System;

using marsroverprob.commands.intf;
using marsroverprob.model;
using marsroverprob.helper;

namespace marsroverprob.commands.rotateleft 
{
    class RotateLeftCommand : IRoverCommand {
        public void execute(Rover rover, String maxPos) {
            RoverHelper.rotateLeft(rover, maxPos);
        }
    }
}